src/test/java/com/training/services/tests/LoyalityUpdateTest.java

```diff
@@ -74,4 +71,136 @@ public void testLoyality() {
         softAssert.assertThat(loyaltyResponse.getPayload().getRelationshipPoints()).isEqualTo("83")
                 .as("Relationship points is not equal to 83");
     }
+
+    /**
+     * Putting wrong app key, In loyality API, and checking if we are getting correct response from sever
+     */
+    @Test
+    public void testLoyalityGANegativeWrongAppKey() {
+        headerMap.put(map.get("accessToken"), authenticationResponse.getPayload().getAccessToken());
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyWrongValue"));
+
+        com.training.services.ga.loyalty.RequestBody requestBody = new com.training.services.ga.loyalty.RequestBody();
+
+        requestBody.setBrand("R");
+        requestBody.setChannel("web");
+        requestBody.setVdsId(authenticationResponse.getPayload().getAccountId());
+        requestBody.setLastName("Poole");
+        requestBody.setLoyaltyId("137529822");
+        requestBody.setBirthdate("19620802");
```

Comment:
Can we add the setting of the below in beforeAll?
- setBrand
- setChannel
- setLastName
- setBirthdate